src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -90,7 +129,7 @@ public void Remove(PointerPoint removed)
 				if (TryUpdate(removed))
 				{
 					// For now we complete the Manipulation as soon as a pointer was removed ...
-					// not checked the UWP behavior for that!
+					// did not checked the UWP behavior for that!
```

Comment:
typo: did not check

Isn't it something we should validate now?

Reply:
Not really : even if it should work, the support of multi-touch is not yet supported by Uno (except for zoom support in the `ScrollViewer`)